phpBB/phpbb/console/command/style/show.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+namespace phpbb\console\command\style;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class show extends command
+{
+	protected function configure()
+	{
+		$this
+			->setName('style:show')
+			->setDescription($this->user->lang('CLI_DESCRIPTION_LIST_STYLES'))
+		;
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output)
+	{
+		$io = new SymfonyStyle($input, $output);
+
+		$installed = $this->manager->get_installed_styles();
+
+		$enabled = array_filter($installed, create_function('$v', 'return $v[\'style_active\'];'));
```

Comment:
Each PHP statement must be on a line by itself
https://travis-ci.org/phpbb/phpbb/jobs/222678414